charts/retool/templates/_helpers.tpl

```diff
@@ -164,6 +162,31 @@ Usage: (include "retool.workflows.enabled" .)
 {{- $output -}}
 {{- end -}}
 
+{{/*
+Set Code Executor enabled
+Usage: (include "retool.codeExecutor.enabled" .)
+*/}}
+{{- define "retool.codeExecutor.enabled" -}}
+{{- $output := "" -}}
+{{- if or
+    (eq (toString .Values.codeExecutor.enabled) "true")
+    (eq (toString .Values.codeExecutor.enabled) "false")
+-}}
+  {{- if (eq (toString .Values.codeExecutor.enabled) "true") -}}
+    {{- $output = "1" -}}
+  {{- else -}}
+    {{- $output = "" -}}
+  {{- end -}}
+{{- else if empty .Values.codeExecutor.image.tag -}}
+  {{- $output = "" -}}
+{{- else if semverCompare ">= 3.20.15" .Values.codeExecutor.image.tag -}}
```

Comment:
3.20.15 is the latest patch of 3.20, which is when we started shipping code executors with matching tags consistently

Reply:
Is there a way we can automatically set `codeExecutor.image.tag` to the retool version? Could simplify things later on

Reply:
I was thinking that as well, will follow up with that change next